src/utils/fishData.js

```diff
@@ -1448,6 +1479,63 @@ const fishComponents = {
       ]
     }
   },
+  scales: {
+    scales2: {
+      src: scales2_image,
+      knnData: [],
```

Comment:
Will the scales have an impact on the ML?

Reply:
Very slightly. I've added them in as data so if someone chooses all fish with the same scales it should show them fish with those scales. I don't think it will have a negative impact on part 2 now that we use SVM.